scipy/signal/_filter_design.py

```diff
@@ -1083,6 +1083,18 @@ def tf2zpk(b, a):
     transfer function coefficients must first be converted to the "positive
     powers" form before finding the poles and zeros.
 
+    Examples
+    --------
+    Find the zeroes, poles and gain of 
+    a filter with the transfer function
+        
+        H(s) = 3s^2 / (s^2 + 5s + 13)
+        
+    >>> from scipy.signal._filter_design import tf2zpk
```

Comment:
The correct public import is
```suggestion
    >>> from scipy.signal import tf2zpk
```

Reply:
Thank you for the feedback